parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -12,7 +12,9 @@
       "[variables('vnetID')]"
 {{end}}
 {{else}}
-{{if not .IsCustomVNET}}
+{{if .IsCustomVNET}}
```

Comment:
@ritazh looks good, just need to remove the documentation wording about putting the IP address at the end then we're done

Reply:
done